Source: Anders Ossowicki <aowi@exherbo.org>
Reason: Net-Server attempts to bind to 0.0.0.0 when host is unspecified.
Upstream: No (behaviour is intentional).

diff --git a/t/Server_BASE.t b/t/Server_BASE.t
index 8cea573..96b932d 100644
--- a/t/Server_BASE.t
+++ b/t/Server_BASE.t
@@ -128,7 +128,7 @@ if( $fork && $pipe ){
 
       ### start the server
       close STDERR;
-      Net::Server::Test->run(port => $ports[0]);
+      Net::Server::Test->run(host => 'localhost', port => $ports[0]);
       exit;
 
     }
@@ -192,7 +192,8 @@ if( $fork && $pipe){
 
       ### start up a server bound to two ports
       close STDERR;
-      Net::Server::Test->run(port => "localhost:$ports[2]",
+      Net::Server::Test->run(host => 'localhost',
+                             port => "localhost:$ports[2]",
                              port => $ports[1] );
       exit;
 
diff --git a/t/Server_Fork.t b/t/Server_Fork.t
index 01cdc97..0a5e9fe 100644
--- a/t/Server_Fork.t
+++ b/t/Server_Fork.t
@@ -102,7 +102,7 @@ if (! $fork || ! $pipe) {
         if ($pid) {
 
             close STDERR;
-            Net::Server::Test->run(port => $ports[0]);
+            Net::Server::Test->run(host => 'localhost', port => $ports[0]);
             exit;
 
         ### child does the client - wait for accept to signal we are ready
diff --git a/t/Server_INET.t b/t/Server_INET.t
index d6c382e..af8e5e9 100644
--- a/t/Server_INET.t
+++ b/t/Server_INET.t
@@ -128,7 +128,7 @@ if( $fork && $pipe ){
       close STDERR;
 
       ### run the child
-      Net::Server::Test->run(port => $ports[0]);
+      Net::Server::Test->run(host => 'localhost', port => $ports[0]);
       exit;
 
     }
diff --git a/t/Server_MultiType.t b/t/Server_MultiType.t
index 7ce25d2..4bfb5bf 100644
--- a/t/Server_MultiType.t
+++ b/t/Server_MultiType.t
@@ -124,7 +124,8 @@ if( $fork && $pipe ){
     }else{
 
       close STDERR;
-      Net::Server::Test->run(server_type => 'non-existant',
+      Net::Server::Test->run(host => 'localhost',
+                             server_type => 'non-existant',
                              port        => $ports[0]);
       exit;
 
diff --git a/t/Server_Multiplex.t b/t/Server_Multiplex.t
index 6710bec..91aea13 100644
--- a/t/Server_Multiplex.t
+++ b/t/Server_Multiplex.t
@@ -131,7 +131,8 @@ if( $pid ){
 
   $success = 1;
   close STDERR;
-  __PACKAGE__->run(port => $ports[0],
+  __PACKAGE__->run(host => 'localhost',
+                   port => $ports[0],
                    setsid => 1,
                    );
   exit;
diff --git a/t/Server_PreFork.t b/t/Server_PreFork.t
index 22818ea..4b1197b 100644
--- a/t/Server_PreFork.t
+++ b/t/Server_PreFork.t
@@ -149,7 +149,8 @@ if( $fork && $pipe){
 
       ### start up a server bound to two ports
       close(STDERR);
-      Net::Server::Test->run(port => "localhost:$ports[0]",
+      Net::Server::Test->run(host => 'localhost',
+                             port => "localhost:$ports[0]",
                              port => $ports[1],
                              setsid => 1,
                              min_servers  => 1,
diff --git a/t/Server_PreForkSimple.t b/t/Server_PreForkSimple.t
index ec0495f..e216966 100644
--- a/t/Server_PreForkSimple.t
+++ b/t/Server_PreForkSimple.t
@@ -149,7 +149,8 @@ if( $fork && $pipe){
 
       ### start up a server bound to two ports
       close(STDERR);
-      __PACKAGE__->run(port => "localhost:$ports[0]",
+      __PACKAGE__->run(host => 'localhost',
+                             port => "localhost:$ports[0]",
                              port => $ports[1],
                              setsid => 1,
                              max_servers  => 2,
diff --git a/t/UDP_test.t b/t/UDP_test.t
index 119ee08..3edcb84 100644
--- a/t/UDP_test.t
+++ b/t/UDP_test.t
@@ -1,5 +1,4 @@
-BEGIN { $| = 1; print "1..5\n"; }
-
+use Test::More skip_all => "Sydbox violations";
 
 ### load the module
 END {print "not ok 1\n" unless $loaded;}
diff --git a/t/UNIX_test.t b/t/UNIX_test.t
index e5ffdff..9a776fd 100644
--- a/t/UNIX_test.t
+++ b/t/UNIX_test.t
@@ -1,6 +1,4 @@
-BEGIN { $| = 1; print "1..5\n"; }
-
-
+use Test::More skip_all => "Sydbox violations";
 ### load the module
 END {print "not ok 1\n" unless $loaded;}
 use Net::Server;
-- 
1.7.0.2

